README.md

```diff
@@ -30,10 +30,81 @@ How to run it
 -------------
 To preview patch generated test cases:
 ```bash
-$ kpet --db . run print-test-cases 001.patch
+$ kpet --db <path-to-kpet-db> run print-test-cases 001.patch
 ```
 
 To generate complete beaker xml:
 ```bash
-$ kpet --db . run generate --description 'skt ##KVER##' -a aarch64 -k '##KPG_URL##' -t upstream 001.patch
+$ kpet --db <path-to-kpet-db> run generate --description 'skt ##KVER##' -a aarch64 -k '##KPG_URL##' -t upstream 001.patch
 ```
+
+Exported variables for kpet-db templates
+----------------------------------------
+The following variables are passed to the Beaker templates in
+kpet-db. They effectively are the interface between kpet and whichever
+database you decide to use:
+
+* `DESCRIPTION`: Description for the Beaker test run.
+* `KURL`: URL to the tarball containing the kernel to be tested.
```

Comment:
This is a rather unfortunate name for the variable and needs to be fixed. Right now it's just an opaque kernel location, and in practice could be a tarball URL, or a special yum repo URL, maybe something else as well. Previously it was a special string to be replaced by skt. So, it needs a more general description.